uniffi_bindgen/src/bindings/gecko/templates/WebIDLTemplate.webidl

```diff
@@ -0,0 +1,69 @@
+// This file was autogenerated by some hot garbage in the `uniffi` crate.
+// Trust me, you don't want to mess with it!
+
+{%- for rec in ci.iter_record_definitions() %}
+dictionary {{ rec.name()|class_name_webidl }} {
+  {%- for field in rec.fields() %}
+  required {{ field.type_()|type_webidl }} {{ field.name()|var_name_webidl }};
+  {%- endfor %}
+};
+{% endfor %}
+
+{%- for e in ci.iter_enum_definitions() %}
+enum {{ e.name()|class_name_webidl }} {
+  {% for variant in e.variants() %}
+  "{{ variant|enum_variant_webidl }}"{%- if !loop.last %}, {% endif %}
+  {% endfor %}
+};
+{% endfor %}
+
+{%- let functions = ci.iter_function_definitions() %}
+{%- if !functions.is_empty() %}
+[ChromeOnly, Exposed=Window]
+namespace {{ ci.namespace()|class_name_webidl }} {
+  {#-
+  // We'll need to figure out how to handle async methods. One option is
+  // to declare them as `async foo()`, or an `[Async]` or `[BackgroundThread]`
+  // attribute in the UniFFI IDL. Kotlin, Swift, and Python can ignore that
+  // anno; Gecko will generate a method that returns a `Promise` instead, and
```

Comment:
Is "anno" short for "annotation"? If so, I've not seen that abbreviation before. Probably best to write the full word.